_data/simple-icons.json

```diff
@@ -3623,6 +3623,11 @@
             "source": "https://www.digitalocean.com/press/",
             "guidelines": "https://www.digitalocean.com/press/"
         },
+        {
+            "title": "Diners Club",
+            "hex": "004C97",
+            "source": "https://www.dinersclub.com/about-us/press/"
```

Comment:
Their media kit isn't working for me - I can however load the SVG you mention, which is different.